changelogs/fragments/2454-detect_zfs_changed.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - zfs - Certain ZFS properties, especially sizes, would lead to a task being falsely marked as "changed" even when no actual change was made
```

Comment:
could add as well a reference to opened issues, like fixes #975, etc..
please see other changelog fragments for examples...

Reply:
I updated the changelog to reference that bug and added a link to this PR. Let me know if it should be formatted differently though.